uber/site_sections/registration.py

```diff
@@ -890,6 +890,26 @@ def watch_list(self, session, message='', **params):
             'message': message
         }
 
```

Comment:
Yeah, this won't be going back into master for sure.

The advantage of doing it this way is that we were able to be sure that the fix would reflect exactly what it would have done if the code had been right in the first place, which works both to make sure it's done correctly and to make sure we actually fixed the bug properly (you'll see that in one of these PR I had to adjust the logic because it wasn't inclusive enough).

Also in this case, we had almost 5000 badges to fix, so I wasn't too keen on just doing it via ssh, haha.